lua/conform/runner.lua

```diff
@@ -359,40 +363,28 @@ local function run_formatter(bufnr, formatter, config, ctx, input_lines, opts, c
   if env then
     log.debug("Run ENV: %s", env)
   end
-  local stdout
-  local stderr
   local exit_codes = config.exit_codes or { 0 }
   local jid
-  local ok, jid_or_err = pcall(vim.fn.jobstart, cmd, {
-    cwd = cwd,
-    env = env,
-    stdout_buffered = true,
-    stderr_buffered = true,
-    stdin = config.stdin and "pipe" or "null",
-    on_stdout = function(_, data)
-      if config.stdin then
-        stdout = data
-      end
-    end,
-    on_stderr = function(_, data)
-      stderr = data
-    end,
-    on_exit = function(_, code)
+  local ok, job_or_err = pcall(
+    vim.system,
+    cmd,
```

Comment:
One of the less-documented (possibly undocumented?) features of conform is that if `args` is a string or function that returns a string (as opposed to the normal `string[]`), the `build_cmd` function above will _also_ return a string. Passing a string to `jobstart` will run the command using the shell instead of executing it directly, allowing you to use pipes and other shell features. `vim.system` only accepts a `string[]`, so we'll need to construct that ourselves.

Reply:
I've changed the `build_cmd` function to always return `string[]` to run the shell command if required.